### PR TITLE
Update translations guiSQLiteStudio_zh_CN.ts (Chinese Simplified)

### PR DESCRIPTION
**Pull request by jiangzc**

### CODE REVIEW

File: SQLiteStudio3/guiSQLiteStudio/translations/guiSQLiteStudio_zh_CN.ts [hunk 1]
```diff
@@ -558,7 +558,8 @@
     <message>
       <location filename="../constraints/columngeneratedpanel.ui" line="71"/>
       <source>Named constraint:</source>
-      <translation>已命名的约束：</translation>
```
Comment (pawelsalawa) [inline SQLiteStudio3/guiSQLiteStudio/translations/guiSQLiteStudio_zh_CN.ts:561]: This doesn't seem right. The actual comment is moved into the <translatorcomment> tag, while the translation is reversed to the English version. Is this by a mistake?

File: SQLiteStudio3/guiSQLiteStudio/translations/guiSQLiteStudio_zh_CN.ts [hunk 2]
```diff
@@ -2167,7 +2169,8 @@ All objects from this group will be moved to parent group.</source>
     <message>
       <location filename="../dbtree/dbtree.cpp" line="1705"/>
       <source>Remainig objects from deleted group will be moved in place where the group used to be.</source>
-      <translation type="unfinished">Remainig objects from deleted group will be moved in place where the group used to be.</translation>
+      <translatorcomment>被删除分组中的剩余子对象会移动到原来的位置</translatorcomment>
+      <translation>Remainig objects from deleted group will be moved in place where the group used to be.</translation>
```
Comment (pawelsalawa) [inline SQLiteStudio3/guiSQLiteStudio/translations/guiSQLiteStudio_zh_CN.ts:2173]: The <translation> here is still untranslated, whereas the translated text seems to be placed in the <translatorcomment> field. I believe this is also by mistake?

**Review by pawelsalawa — CHANGES_REQUESTED** [review]
I found 2 places there the translation was placed in the comment field, instead of the translation. It looks to be by mistake. Can you please fix it, or otherwise explain why it should be like this?

**Review by pawelsalawa — APPROVED** [review]

**Comment by jiangzc** [comment]
It is a mistake and I fixed it in the next commit .  Thanks for your code review.

**Pull request merged**